src/pages/edge/edge-network/tabs/api-reference.md

```diff
@@ -582,9 +898,47 @@ Map<String, Object> data = new HashMap<>();
 data.put("free", "form");
 data.put("data", "example");
 
+// ----------------- Datastream config overrides map start -----------------
+Map<String, Object> configOverrides = new HashMap<>();
+
+// com_adobe_experience_platform
+Map<String, Object> experiencePlatform = new HashMap<>();
+Map<String, Object> datasets = new HashMap<>();
+
+Map<String, Object> eventDataset = new HashMap<>();
+eventDataset.put("datasetId", "SampleEventDatasetIdOverride");
+
+Map<String, Object> profileDataset = new HashMap<>();
+profileDataset.put("datasetId", "SampleProfileDatasetIdOverride");
+
+datasets.put("event", eventDataset);
+datasets.put("profile", profileDataset);
+
+experiencePlatform.put("datasets", datasets);
+configOverrides.put("com_adobe_experience_platform", experiencePlatform);
+
+// com_adobe_analytics
+Map<String, Object> analytics = new HashMap<>();
+analytics.put("reportSuites", new String[]{"rsid1", "rsid2", "rsid3"});
+configOverrides.put("com_adobe_analytics", analytics);
+
+// com_adobe_identity
+Map<String, Object> identity = new HashMap<>();
+identity.put("idSyncContainerId", "1234567");
+configOverrides.put("com_adobe_identity", identity);
+
+// com_adobe_target
+Map<String, Object> target = new HashMap<>();
+target.put("propertyToken", "SamplePropertyToken");
+configOverrides.put("com_adobe_target", target);
+// ----------------- Datastream config overrides map end -----------------
+
+
```

Comment:
This seems repeating info, I am thinking we can skip these examples in the class definition since we are covering it with sendEvent API reference.